nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c

```diff
@@ -1512,7 +1518,9 @@ ProcConvertSelection(register ClientPtr client)
 					CurrentSelections[i].pWin))
 #endif
 	    )
+
 	{        
+	    memset(&event, 0, sizeof(xEvent));
```

Comment:
I'd just rebase this like you have done in `nx-X11/programs/Xserver/dix/dispatch.c`: move this up to line 1480, drop here and on line 1537.